WordPress/src/main/java/org/wordpress/android/modules/ExperimentModule.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.modules
+
+import dagger.Module
+import dagger.multibindings.Multibinds
+import org.wordpress.android.util.experiments.Experiment
+
+@Module
+interface ExperimentModule {
+    @Multibinds fun experiments(): Set<Experiment>
+
+    // Copy and paste the line below to add a new experiment.
+    // @Binds @IntoSet fun exampleExperiment(experiment: ExampleExperiment): Experiment
```

Comment:
Is it viable to introduce some kind of a lint rule that would fail if there is an experiment that is not registered in the module? Maybe that's unnecessary.

Reply:
This is probably viable, although I have no experience with custom lint rules so I can't say for sure.

That said, if that's something we want to do, we should probably do it in a different PR (specially since I guess it would need to be submitted to [WordPress-Lint-Android](https://github.com/wordpress-mobile/WordPress-Lint-Android)?).